lib/stat_tracker.rb

```diff
@@ -36,6 +36,14 @@ def percentage_home_wins
 
     (home_wins / self.total_games).round(2)*100
   end
+  
+  def percentage_visitor_wins
+    visitor_wins = @game_teams.count do |game|
+      game.result == "WIN" && game.hoa == "visitor"
```

Comment:
I wonder if for readability we can use something other than visitor? Could we use the data from the games data to compare team perhaps?